draft-ietf-quic-transport.md

```diff
@@ -2740,7 +2697,57 @@ that it sends.
 Strategies and implications of the frequency of generating acknowledgments are
 discussed in more detail in {{QUIC-RECOVERY}}.
 
-## Special Considerations for PMTU Discovery
+
+## Packet Size
+
+Clients MUST ensure that the first packet in a connection, and any
+retransmissions of those octets, has a QUIC packet size of least 1200 octets.
+The packet size for a QUIC packet includes the QUIC header and integrity check,
```

Comment:
s/"packet size for QUIC packet"/"QUIC packet size"

Keep the terminology consistent, and shorter

Reply:
I don't think we need this; "QUIC packet size" is defined in Section 2.

Reply:
I never realized that "QUIC packet size" was defined there.  I think that this is the better place to add that definition.